fabric-convention-tags-v1/src/generated/resources/data/c/tags/worldgen/biome/climate_cold.json

```diff
@@ -7,9 +7,11 @@
     "minecraft:snowy_taiga",
     "minecraft:old_growth_spruce_taiga",
     "minecraft:old_growth_pine_taiga",
+    "minecraft:snowy_plains",
+    "minecraft:ice_spikes",
```

Comment:
Don't edit this file manually. It's generated automatically (see build.gradle file of the module). Please update the code and re-run datagen instead.